src/main/storage-measure.test.ts

```
@@ -0,0 +1,82 @@
+		const result = await measureStorage(filesystemPath, legacyPath);
+		assert.deepEqual(
+			result.entries.sort((a, b) => a.name.localeCompare(b.name)),
+			[
+				{ name: 'orders', bytes: 10, root: 'fsdbs' },
+				{ name: 'plain.bin', bytes: 3, root: 'fsdbs' },
+				{ name: 'store_v3.sqlite3', bytes: 5, root: 'legacy-sqlite' },
+			]
+		);
+
+		assert.deepEqual(
+			await measureStorage(path.join(fixture, 'missing-fsdbs'), path.join(fixture, 'missing-legacy')),
```

ESLint

[error] Replace `path.join(fixture,·'missing-fsdbs'),·path.join(fixture,·'missing-legacy')` with `⏎↹↹↹↹path.join(fixture,·'missing-fsdbs'),⏎↹↹↹↹path.join(fixture,·'missing-legacy')⏎↹↹↹`

(prettier/prettier)